controllers/comments-controller.js

```diff
@@ -0,0 +1,20 @@
+const {
+  selectCommentsByArticleId,
+  insertCommentsByArticleId,
+} = require("../models/comments-model");
+
+exports.getCommentsByArticleId = (req, res, next) => {
+  selectCommentsByArticleId(req.params)
+    .then((comments) => {
+      res.status(200).send({ comments });
+    })
+    .catch((err) => next(err));
+};
+
+exports.postCommentByArticleId = (req, res, next) => {
+  insertCommentsByArticleId(req.body, req.params)
+    .then(() => {
+      res.status(201).send();
```

Comment:
You are able to get the comment back from the insertComments.  You should send back the comment created back and test it has the correct properties.